### PR TITLE
Monetization (SEO-77): HtmlTitle - add tests for mercury

### PR DESCRIPTION
**Pull request by saipetch**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* INFO 1 info

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* INFO [Complete the task associated to this TODO comment.](https://github.com/Wikia/selenium-tests/blob/88c14010a258a40fbc18f7b52073e4d0499284eb/src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java#L17) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)

File: src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java [hunk 1]
```diff
@@ -12,6 +12,7 @@
 /**
  * @ownership Seo
  */
+@Test(groups = {"Seo", "SeoHtmlTitle"})
 public class HtmlTitleTests extends NewTestTemplate {
 
   // TODO: QAART-681 Sonar should not report duplicate strings warnings for data providers
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java]: INFO Complete the task associated to this TODO comment. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* INFO 1 info

Watch the comments in this conversation to review them.

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**